Wikipedia/Localizations/pl.lproj/Localizable.strings

```diff
@@ -39,6 +39,8 @@
 "about-libraries-complete-list" = "Pełna lista";
 "about-libraries-license" = "Licencja";
 "about-libraries-licenses-title" = "Kochamy otwartoźródłowe oprogramowanie $1";
+"about-places-maps-license" = "Licencja na mapy miejsc";
+"about-places-maps-license-details" = "Miejsca korzystają z map dostarczonych przez Apple Maps 1$";
```

Comment:
This wasn't generating the native localization - hopefully [flipping](https://translatewiki.net/w/i.php?title=Wikimedia%3AWikipedia-ios-about-places-maps-license-details%2Fpl&type=revision&diff=8969737&oldid=8964869) to $1 will work.